config/webpack.prod.config.js

```diff
@@ -144,6 +144,7 @@ module.exports = Merge.smart(commonConfig, {
       ACCESS_TOKEN_COOKIE_NAME: null,
       USER_INFO_COOKIE_NAME: null,
       CSRF_COOKIE_NAME: 'csrftoken',
+      LANGUAGE_PREFERENCE_COOKIE_NAME: null,
```

Comment:
Really?

Reply:
They get overridden by edx-internal variables (not sure exactly what this process is)

Reply:
I imagine it will need to get added to the prod/stage files here:
https://github.com/edx/edx-internal/blob/29a43bc351b18621ff8dbe3d54dec0103d0619a5/frontends/common/prod_config.yml

Reply:
We're all set there